internal/exec/runner.go

```diff
@@ -326,13 +324,13 @@ func (r *runner) ProtocCommands(args []string, genCommands bool) error {
 	return nil
 }
 
-func (r *runner) Lint(args []string) error {
+func (r *runner) Lint(args []string, dryRun bool) error {
 	meta, err := r.getMeta(args)
 	if err != nil {
 		return err
 	}
 	r.printAffectedFiles(meta)
-	if _, err := r.compile(false, false, meta); err != nil {
+	if _, err := r.compile(false, false, dryRun, meta); err != nil {
```

Comment:
`dry-run` doesn't really apply for `lint` and `format` - they're just going to print out the commands from `compile`, it's not printing out the dry run of what it's linting, which will be confusing (it will just print the `protoc` commands from `compile`, but then does more stuff). I would remove for `lint` and `format`.

Reply:
Yea, sounds reasonable. I'll update!